core/src/main/scala/org/apache/spark/internal/Logging.scala

```diff
@@ -135,7 +135,8 @@ private[spark] trait Logging {
         val replLevel = Option(replLogger.getLevel()).getOrElse(Level.WARN)
         if (replLevel != rootLogger.getEffectiveLevel()) {
           System.err.printf("Setting default log level to \"%s\".\n", replLevel)
-          System.err.println("To adjust logging level use sc.setLogLevel(newLevel).")
+          System.err.println("To adjust logging level use sc.setLogLevel(newLevel). " +
+          "For SparkR, use setLogLevel(newLevel).")
```

Comment:
While I'd indent this continuation line, I don't think it's worth worrying about. LGTM